src/main/io/displayport_msp_bf_compat.c

```diff
@@ -192,13 +193,13 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 */
     case SYM_WATT:
         return BF_SYM_WATT;
-/*
+
     case SYM_MW:
-        return BF_SYM_MW;
+        return 'M' + 'W';
```

Comment:
You can't do this. Symbols are a single space character. Adding multiple character will cause issues with the OSD creation.